ras_party/views/respondent_view.py

```diff
@@ -38,3 +38,10 @@ def get_respondent_by_id(id):
 def get_respondent_by_email(email):
     response = ras_party.controllers.respondent_controller.get_respondent_by_email(email)
     return make_response(jsonify(response), 200)
+
+
+@respondent_view.route('/respondents/change_respondent_details', methods=['PUT'])
```

Comment:
I feel like the endpoint should be `/respondents/change_respondent_details/<respondent_id>` with `respondent_id` in the url rather than the payload.

Reply:
yes def agree with this